src/main/cms/cms_menu_imu.c

```diff
@@ -391,6 +391,73 @@ static CMS_Menu cmsx_menuFilterPerProfile = {
     .entries = cmsx_menuFilterPerProfileEntries,
 };
 
+static uint8_t cmsx_dstPidProfile;
```

Comment:
OK, @mikeller suggested in an earlier comment to use this condition for both. I will change it in a unique condition for all the code related to this.